docs/src/design/decoder/main.md

```diff
@@ -25,8 +25,8 @@ Managing control flow in the VM is accomplished by executing control flow operat
 | `SPLIT`   | Initiates processing of a new [Split block](../programs.md#split-block).     |
 | `LOOP`    | Initiates processing of a new [Loop block](../programs.md#loop-block).       |
 | `REPEAT`  | Initiates a new iteration of an executing loop.                              |
-| `SPAN`    | Initiates processing of a new [Span block](../programs.md#span-block).       |
-| `RESPAN`  | Initiates processing of a new operation batch within a span block.           |
+| `SPAN`    | Initiates processing of a new [Basic block](../programs.md#basic-block). (historically called "span block") |
+| `RESPAN`  | Initiates processing of a new operation batch within a basic block. (historically called "span block") |
```

Comment:
I wonder if at some point we'd want to rename `SPAN` and `RESPAN` into something else - but I don't have good suggestions for now.